### PR TITLE
Added Livestream to the services json

### PR DESCRIPTION
**Pull request by neogenix**
Added Livestream to the services json

### CODE REVIEW

File: plugins/rtmp-services/data/services.json [hunk 1]
```diff
@@ -842,5 +842,14 @@
                 "max audio bitrate": 160
             }
         }
```
Comment (notr1ch) [inline plugins/rtmp-services/data/services.json]: There's a missing comma here and instead it is at the bottom of the new section. This makes the file invalid JSON.

Comment (neogenix) [inline plugins/rtmp-services/data/services.json]: Sorry my bad. I'll fix that shortly.

Comment (neogenix) [inline plugins/rtmp-services/data/services.json]: @notr1ch should be fixed in the PR now. sorry about the lag.

**Comment by Nightfirecat** [comment]
> package.json was also not updated.

This is still an issue.

**Comment by neogenix** [comment]
@Nightfirecat oh, wow, I'm a moron. Sorry, I was focused on the bad juju in the services.json, not the package.json. I'll update that shortly.

**Comment by jp9000** [comment]
This should have been one commit.  It also did not follow the commit guidelines.  Read the "contributing" document in the repository root next time if possible.  I'll squash/fix and merge.  Thanks

**Pull request closed without merging**